platform/node/scripts/after_success.sh

```diff
@@ -4,7 +4,7 @@ set -e
 set -o pipefail
 
 if [[ -n ${PUBLISH:-} ]]; then
-    if [[ "${BUILDTYPE}" == "Release" ]]; then
+    if [[ "${BUILDTYPE}" == "RelWithDebInfo" ]]; then
```

Comment:
True. What I am concerned about is the fallback to Debug. How about this instead:

```
if [[ -n ${PUBLISH:-} ]]; then
    if [[ "${BUILDTYPE}" == "RelWithDebInfo" ]]; then
        ./node_modules/.bin/node-pre-gyp package publish info
    elif [[ "${BUILDTYPE}" == "Debug" ]]; then
        ./node_modules/.bin/node-pre-gyp package publish info --debug
    else
        echo "error: must provide either Debug or RelWithDebInfo for BUILDTYPE"
        exit 1
    fi
fi
```